metadata/v1.9/step_3_upgrade_framework.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework to v1.9",
```

Comment:
we should call out the max txn gas decrease here too, corresponding to the step 1 increase

Reply:
To call it out, I added "and decrease max txn gas back to the default value".